modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -28,7 +28,7 @@ import {
 
 import { TestBitGo } from '@bitgo/sdk-test';
 import { BitGo } from '../../../src';
-import { bip32 } from '@bitgo/utxo-lib';
+import { bip32, networks, bitgo as utxoBitgo } from '@bitgo/utxo-lib';
```

Comment:
nit: we usually import this as `utxolib`

Reply:
Thanks, I actually moved this test out of bitgo root module and added to the sdk-coin-btc class since we really just want to test that `postProcessPrebuild` does not decorate the txn with a locktime other than `0`